staging/src/github.com/kubeedge/api/apis/componentconfig/cloudcore/v1alpha1/types.go

```diff
@@ -323,6 +326,9 @@ type EdgeControllerBuffer struct {
 
 // EdgeControllerLoad indicates the EdgeController load
 type EdgeControllerLoad struct {
+	// ProcessEventWorkers indicates the load of process event workers
+	// default 1
```

Comment:
```suggestion
	// default 4
```
according your code, the default value should be 4